internal/pkg/config/import_services.go

```diff
@@ -1,5 +1,16 @@
 package config
 
-type ImportedServices struct {
+import (
+	istionetv1alpha3 "istio.io/api/networking/v1alpha3"
+)
+
+type ImportedServiceSet struct {
 	Rules []Rules `yaml:"rules"`
 }
+
+type ImportedService struct {
+	AppName         string
+	ServiceHostname string
+	ServicePorts    []*istionetv1alpha3.ServicePort
+	Endpoints       []Remote
```

Comment:
Currently, we allow to specify single remote peer in the input configuration, so this will never be an array of remote peers. However, I agree that we want to support more than one remote peers, so we should also change the configuration structure.

Reply:
Beside that, I would prefer decouple imported service info and remote peer specification. In my opinion, `MakeImportedMCPResources` should take imported service and remote peer spec as separate arguments.

Reply:
Ah yes I see, so I suppose we would instead just keep a list of remote names that expose the service in the service spec and retrieve the remote endpoint spec from the second input as appropriate?